sensirion_common.h

```diff
@@ -44,7 +44,7 @@ extern "C" {
 #define NULL ((void *)0)
 #endif
 
-#ifdef BIG_ENDIAN
+#if SENSIRION_BIG_ENDIAN
```

Comment:
use #ifdef and not just #if since its not a boolean comparison

Reply:
But I want it to be boolean. It's in accordance to the snippet code from this commit, where `#define SENSIRION_BIG_ENDIAN 0` is legal. I could change to `if defined(SENSIRION_BIG_ENDIAN) && SENSIRION_BIG_ENDIAN` but I don't think it's wise to assume a default endianness.

Reply:
ok I thought you just wanted a drop-in replacement for BIG_ENDIAN

Reply:
Well, there's still an (explicit) default-endianness because the code ships a default implementation `#define SENSIRION_BIG_ENDIAN 0`